fsh/ig-data/input/pages/3_howtoJoin.md

```diff
@@ -39,6 +39,15 @@ To fill the Store of the Bridgehead you need access to clinical data from your *
 The Store of the Bridgehead stores data as **HL7 FHIR® Bundles**. To import data into the Store, they must be harmonised and converted. Each site must build an ETL-process (extract, transform, load) to harmonise the structure of the data and translate them to fit predefined FHIR® profiles.
```

Comment:
ETL pipeline might be more accurate

Reply:
I would not say:
>The Store of the Bridgehead stores data as **HL7 FHIR® Bundles**.
The bundle is only a transport vehicle. The individual FHIR resources of the bundle are stored, not the bundle itself. That is an important difference, because one could also store bundles explicitly, but we don't do that.